com.unity.ml-agents/Runtime/Actuators/ActuatorComponent.cs

```diff
@@ -12,8 +13,21 @@ public abstract class ActuatorComponent : MonoBehaviour
         /// Create the IActuator.  This is called by the Agent when it is initialized.
         /// </summary>
         /// <returns>Created IActuator object.</returns>
+        [Obsolete("Use CreateActuators instead.")]
```

Comment:
since this is abstract people will still need to override it until we remove it.  The migration doc tells them how to disable the warnings.

Reply:
Ugh, this is annoying but I'm not sure if not marking it obsolete is worse.

Reply:
yeah, i went through the same thought process and ended here.